ckanext/saml2auth/spconfig.py

```diff
@@ -6,6 +6,10 @@
 
 
 def get_config():
+    """ Get the config
+        Read: https://pysaml2.readthedocs.io/en/latest/howto/config.html
+        """
+    
```

Comment:
Remove whitespace. Make sure you run pep8 checks before pushing.

Reply:
Done @mbocevski

Reply:
@mbocevski can we enable pep8 checks as part of CI and then enforced with GH protected branches? That way you can focus your review on the important stuff and leave things like whitespace checks to the computer :)

Reply:
> can we enable pep8 checks as part of CI

@adborden it already is https://github.com/keitaroinc/ckanext-saml2auth/blob/main/bin/travis-run.sh#L5 :) The travis CI checks failed, I was just pointing it out as part of the review :) Happy new year!